db/migrate/20190401225337_create_orders.rb

```diff
@@ -0,0 +1,9 @@
+class CreateOrders < ActiveRecord::Migration[5.1]
+  def change
+    create_table :orders do |t|
+      t.integer :status
```

Comment:
Should the status of an order be pending as default? ie. default: 0 since an order will only be created when an order is placed?

Reply:
Just kidding, found in the user stories that they are able to have an order without placing it.

Reply:
Based on User Story 26, I think it's okay to default all newly created orders to status 0 (pending with an enum that still needs implemented). I cannot think of a situation where we would want a new order to start with any other status.

And I click the button or link to check out
An order is created in the system, which has a status of "pending"

Order Status
'pending' means a user has placed items in a cart and "checked out" to create an order, merchants may or may not have fulfilled any items yet

Reply:
I agree on this, I think default should be 0.

Reply:
Updated the status with default: 0.  Thanks.